component-samples/scripts/keys_gen.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash
+#
+# Copyright 2021 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+shopt -s extglob
+set -e
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+components=("manufacturer" "owner" "reseller")
+COMP_PATH=$1
+CERTS_PATH=$COMP_PATH/creds
+MFG_CERTS_PATH=$COMP_PATH/${components[0]}
+OWN_CERTS_PATH=$COMP_PATH/${components[1]}
+RES_CERTS_PATH=$COMP_PATH/${components[2]}
+
+usage()
+{
+        echo -e "Usage:
+        $0  <absolute_path_of_pri_components>  <OPTIONS>\n
```

Comment:
absolute_path_of_pri_components => /path/to/root/of/pri/binaries

Reply:
Updated